man/systemd.device.xml

```diff
@@ -152,9 +151,135 @@
+        This controls the third field of the <literal>io.cost.model</literal> control group attribute. Note that the second field is model, which is set to linear.
+        For details about this control group attribute, see <ulink url="https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html#io-interface-files">IO Interface Files</ulink>
+        </para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>IOCostModelRSeqIops=</varname></term>
+        <listitem><para> Maximum sequential read IOs per second. Takes a value between 0 and <constant>UINT64_MAX</constant>. Defaults to 0.
+        This controls the fourth field of the <literal>io.cost.model</literal> control group attribute. For details about this control group attribute,
+        see <ulink url="https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html#io-interface-files">IO Interface Files</ulink>
+        </para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>IOCostModelRRandIops=</varname></term>
+        <listitem><para> Maximum random read IOs per second. Takes a value between 0 and <constant>UINT64_MAX</constant>. Defaults to 0.
+        This controls the fifth field of the <literal>io.cost.model</literal> control group attribute. For details about this control group attribute,
+        see <ulink url="https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html#io-interface-files">IO Interface Files</ulink>
+        </para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>IOCostModelWbps=</varname></term>
+        <listitem><para> Maximum sequential write IO throughput in bytes per second. Takes a value between 0 and <constant>UINT64_MAX</constant>. Defaults to 0.
+        This controls the sixth field of the <literal>io.cost.model</literal> control group attribute. For details about this control group attribute,
+        see <ulink url="https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html#io-interface-files">IO Interface Files</ulink>
+        </para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>IOCostModelWSeqIops=</varname></term>
+        <listitem><para> Maximum sequential write IOs per second. Takes a value between 0 and <constant>UINT64_MAX</constant>. Defaults to 0.
+        This controls the seventh field of the <literal>io.cost.model</literal> control group attribute. For details about this control group attribute,
+        see <ulink url="https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html#io-interface-files">IO Interface Files</ulink>
+        </para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>IOCostModelWRandIops=</varname></term>
+        <listitem><para> Maximum random write IOs per second. Takes a value between 0 and <constant>UINT64_MAX</constant>. Defaults to 0.
```

Comment:
this shouldn't say UINT64_MAX, if the literal string UINT64_MAX cannot be specified. Say 18446744073709551615 instead.